manifest.php

```diff
@@ -19,17 +19,17 @@
  *
  */
 use oat\taoDeliveryRdf\install\RegisterDeliveryFactoryService;
-use oat\taoDeliveryRdf\install\RegisterDeliveryPublishingService;
 use oat\taoDeliveryRdf\scripts\install\OverrideRuntime;
 use oat\taoDeliveryRdf\scripts\install\RegisterDeliveryAssemblyWrapperService;
 use oat\taoDeliveryRdf\scripts\install\SetUpQueueTasks;
+use oat\taoDeliveryRdf\scripts\install\RegisterFileSystem;
 
 return array(
   'name'        => 'taoDeliveryRdf',
   'label'       => 'Delivery Management',
   'description' => 'Manages deliveries using the ontology',
   'license'     => 'GPL-2.0',
-  'version'     => '4.14.0',
+  'version'     => '5.0.0',
```

Comment:
does the breaking change impact the package format itself or only the API ?

Reply:
Nope only the API